src/ast/passes/codegen_llvm.cpp

```diff
@@ -305,7 +305,7 @@ void CodegenLLVM::kstack_ustack(const std::string &ident,
         b_.CreateGEP(stack_struct, buf, { b_.getInt64(0), b_.getInt32(1) }));
     // store pid
     b_.CreateStore(
-        b_.CreateTrunc(b_.CreateGetPidTgid(loc), b_.getInt32Ty()),
+        b_.CreateGetPid(loc),
```

Comment:
The old code here used to extract `tid`, not `pid`, despite the comment. I don't understand why it'd need to be `tid` instead of `pid,` so I think it was just a mistake right from the beginning: 6f5b3fa2798b79acaebc7b3a77b6a6723e7ba48a